### DISCUSSION TITLE
Can't resolve 'fs' in tailwindcss v2.2.19

### DISCUSSION THREAD

**Discussion by thamibn** [tailwindlabs/tailwindcss#5962] [Help]
I am getting this error

ERROR in ./node_modules/tailwindcss/lib/plugins/preflight.js 8:33-46
Module not found: Error: Can't resolve 'fs' in '/home/thamsanca/Documents/Projects/twinsaver_portal/node_modules/tailwindcss/lib/plugins'

not sure what happend it was working fine before

what might i be doing wrong here.

I am using laravel with tailwindcss

here is my tailwind config
```
const defaultTheme = require('tailwindcss/defaultTheme');
module.exports = {
    purge: [
        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php',
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php',
        './resources/views/*.blade.php',
        './resources/js/**/*.vue',
        './resources/js/*.vue',
        './resources/js/**/*.js',
        './resources/js/*.js',
        './config/*.php',
    ],
    darkMode: false,
    theme: {
        extend: {
            fontFamily: {
                sans: ['Nunito', ...defaultTheme.fontFamily.sans],
            },
            colors:{
                primary: '#519D2B',
                secondary: '#94c11c',
                success: '#10B981',
                warning: '#F59E0B',
                danger: '#EF4444',
                info: '#3B82F6'
            }
        },
    },

    variants: {
        extend: {
            opacity: ['disabled'],
            backgroundColor: ['disabled'],
            textColor: ['disabled'],
        },
    },
    plugins: [
        require('@tailwindcss/forms')
    ]
};
```

**Comment by valkedev** [comment]
Have same probleme here. Any workaround?

**Comment by GJVothel** [comment]
Same here. @valkedev, @thamibn did one of you ever work it out?

**Reply by valkedev** [reply]
yeah but don't really remember how. Try to remove `node_modules` dir and reinstall all modules

**Reply by GJVothel** [reply]
Thanks! But isn't that going to install the same when I don't change my package.json?

**Reply by valkedev** [reply]
yes but it might help

**Reply by GJVothel** [reply]
Unfortunately, it didn't but thank you anyway!

**Reply by GJVothel** [reply]
For anyone wondering, I removed node_modules from the directory and tailwindcss from the package.json ->`npm install` ->`npm install -D tailwindcss` then ran my watcher and it showed me the underlying cause.
